backend/src/main/java/ai/verta/modeldb/entities/metadata/MetadataParameterMappingEntity.java

```diff
@@ -1,85 +0,0 @@
-package ai.verta.modeldb.entities.metadata;
```

Comment:
was this added by @evgeniilezhnin , if so is he fine removing this?

Reply:
yes, it supposed to be moved to MetadataPropertyMappingEntity.java, there is no MetadataParameterMappingEntity.java file in the base branch. Maybe merge from the base branch will help?